<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="sampleFileNameLabel">
       <property name="text">
        <string>Sample Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sampleFileNameLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectFileNamePushButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="processPushButton">
       <property name="text">
        <string>Process</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="sampleThicknessLabel">
       <property name="text">
        <string>Sample Thickness (cm)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sampleThicknessLineEdit">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bkgThicknessLabel">
       <property name="text">
        <string>Background Thickness (cm)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bkgThicknessLineEdit">
       <property name="toolTip">
        <string extracomment="Provide thickness of the background in cms"/>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="energyNptsLabel">
       <property name="text">
        <string>Energy Npts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="energyNptsSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="repeatLabel">
       <property name="text">
        <string>Repeat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="repeatSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="normStdSampleLabel">
       <property name="text">
        <string>Normlization Standard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="normStdSampleComboBox">
       <item>
        <property name="text">
         <string>GC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Water</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stdThicknessLabel">
       <property name="text">
        <string>Standard Thickness (cm)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stdThicknessLineEdit">
       <property name="text">
        <string>0.1055</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="xMinMaxLabel">
       <property name="text">
        <string>Xmin:Xmax</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xMinMaxLineEdit">
       <property name="text">
        <string>0.0:1.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="interpLabel">
       <property name="text">
        <string>Interpolation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="interpComboBox">
       <item>
        <property name="text">
         <string>linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>quadratic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cubic</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PlotWidget" name="plotWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>PlotWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
